HISTORY.rst

```diff
@@ -2,6 +2,11 @@
 History
 =======
 
+Current
```

Comment:
I think the version should be updated by you when you decide it is the moment for a new release, mainly because you are the core developer of this library. 

Also, as more PRs will be merged into the ``master`` branch, the changes will be written by the submitters in the ``Current`` and version releases will include multiple changes.

At least that's my view on this. Cheers!